test/example/execute-test.ts

```diff
@@ -0,0 +1,94 @@
+/*
+ * Example tests against a fulfillment's executeHandler
+ */
+/// <reference types="@google/local-home-sdk" />
+/// <reference types="@types/node" />
+import test from 'ava';
+import {
+  extractMockLocalHomePlatform,
+  DeviceManagerStub,
+  extractDeviceManagerStub,
+  MockLocalHomePlatform,
+} from '../../src';
+import {
+  identifyHandler,
+  executeHandler,
+  createUdpDeviceCommand,
+} from './fixtures';
+import {UdpResponseData} from '../platform/test-platform-fixtures';
+
+const DEVICE_ID = 'device-id-123';
+const DEVICE_PORT = 12345;
+const LOCAL_DEVICE_ID = 'local-device-id-123';
+const EXECUTE_REQUEST_ID = 'request-id-123';
+const app: smarthome.App = new smarthome.App('0.0.1');
+
+function createSimpleExecuteCommands(
+  deviceId: string,
+  command: string,
+  params: object
+): smarthome.IntentFlow.ExecuteRequestCommands {
+  return {
+    devices: [{id: deviceId}],
+    execution: [{command, params}],
```

Comment:
most command would rely on customData for dispatching the command should we show an example for that>

Reply:
I added a `customData` parameter and value for this helper, and changed the example test to use a simple `customData` object.